src/pages/Events/events_data.js

```diff
@@ -16,11 +16,31 @@ const events = [
     Time to make all those dreams one step closer to reality! The ASEAN Society Young Entrepreneur Case Competition is aimed at generating real solutions to economic, political, and socio-cultural problems faced by ASEAN businesses. You’ll be contributing to the solution of real issues in Southeast Asia and facilitating discussion of innovative ideas!
     The prizes? A seat in Guru Advantage’s Global Virtual Internship Program (worth over $1000), a Business Leaders Masterclass, personalized letters of commendation from The Australian-ASEAN Business Council, and more!*
     Think you got a winning pitch? Apply now! The internship is also eligible for international and remote students 🤩`,
-    }, 
+  },
+
+  {
```

Comment:
Would be good to move the content somewhere else (like CMS, or backend, or even just some static json file)